src/components/MainComponent.vue

```diff
@@ -5,7 +5,7 @@
         <v-icon>mdi-github</v-icon>
       </v-btn>
 
-      <v-toolbar-title>Copilot Metrics Viewer | {{ capitalizedItemName }} : {{ gitHubOrgName }}</v-toolbar-title>
```

Comment:
when the scope is enterprise, it is better to display the VUE_APP_GITHUB_ENTname here, not the gitHubOrgName